samza-azure/src/main/java/org/apache/samza/AzureConfig.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.samza;
+
+import org.apache.samza.config.Config;
+import org.apache.samza.config.ConfigException;
+import org.apache.samza.config.MapConfig;
+
+
+public class AzureConfig extends MapConfig {
```

Comment:
Can you update the [Samza configuration page](http://samza.apache.org/learn/documentation/latest/jobs/configuration-table.html) with a section for "Samza on Azure" and add config relevant to this component? 
It helps to add the configs incrementally with your PRs because:
1. You can make sure that you don't miss any configs in the end
2. It will be useful for reviewers to understand your code better. For example, it's not obvious to me how `AZURE_BLOB_NAME` and `AZURE_CONTAINER_NAME` is used and when it should be overridden by the user.